src/components/accessibility/screen_reader_status/screen_reader_status.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, {
+  FunctionComponent,
+  ReactNode,
+  useEffect,
+  useState,
+} from 'react';
+
+import { EuiScreenReaderOnly } from '../screen_reader_only';
+
+export interface EuiScreenReaderStatusProps {
+  /**
+   * Use the `id` of the target element when possible
+   */
+  id: string;
+  /**
+   * Whether the make screen readers aware of the content
+   */
+  isActive: boolean;
+  /**
+   * A value to watch to trigger an update so that screen readers pick up changes
+   */
+  updatePrecipitate: number | string;
+  /**
+   * Content for screen readers to announce
+   */
+  children?: ReactNode;
+}
+
+export const EuiScreenReaderStatus: FunctionComponent<EuiScreenReaderStatusProps> = ({
+  id,
+  isActive,
+  updatePrecipitate,
+  children,
+}) => {
+  const [toggle, setToggle] = useState(false);
+
+  useEffect(() => {
+    setToggle((toggle) => !toggle);
+  }, [updatePrecipitate]);
```

Comment:
[follow up to https://github.com/elastic/eui/pull/5157/files#r787135111]

Alternatively instead of a toggle or an `updatePreciptate` prop, why not use `forceRerender` to increment an internal state/key for us whenever the status content changes?

```suggestion
  const forceRender = useForceRender();
  useUpdateEffect(forceRender, [children, forceRender]);
```

Reply:
Because I'm not sure I knew that hook existed 😄

Reply:
Need the current code for the double live region

Reply:
One follow-up question - can we update the toggle status based on `children` changing instead of requiring a separate `updatePrecipitate` prop? I think that will generally be a smoother API for other component usages of this helper, as well as if expose this component publicly as its own component.

```suggestion
  useEffect(() => {
    setToggle((toggle) => !toggle);
  }, [children]);
```

The primary issue I can see with that is if someone wants to repeat the same announcement twice, e.g. ("You have 1 new notification" a minute apart, or "You have 2 results" for different search queries), but I think the solution there is that the user should manually be clearing their status text between events rather than requiring a separate prop. Thoughts?

Reply:
Yeah it's possible I was trying to be to abstract. I'll take a look.